com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -608,8 +608,20 @@ public static List<CommandLineOption> getCommandLineOptions() {
                 opt(null, "debug-output-spirv", ONE, "Force build SPIR-V shaders", false),
                 opt(null, "debug-output-wgsl", ONE, "Force build WGSL shaders", false)
         );
+    }
 
-
+    private static String[] migrateInvalidArguments(String[] args) {
+        ArrayList<String> migratedArguments = new ArrayList<>();
+        for (String arg : args) {
+            // Migrate texture compression argument
+            boolean argIsTextureCompression = arg.startsWith("--tc") || arg.startsWith("--texture-compression");
+            if (argIsTextureCompression && arg.contains("=false")) {
+                System.out.println("Warning: --texture-compression and --tc no longer accepts an argument. To disable texture compression, remove the argument.");
+            } else {
+                migratedArguments.add(arg);
+            }
+        }
+        return migratedArguments.toArray(new String[0]);
     }
```

Comment:
Perhaps add a "DEPRECATED" comment in the code as well, to be able to remove it later when we do a cleanup?